### PR TITLE
bugfixes

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Mantis #0787531
    - Validated if latest connector version 2.0.0 is getting installed by default

Mantis #0795288
    - Rename Sample playbook collection to "SOC Simulator"
    - Added Sample Playbooks in SP

Mantis #0795294
    - Renamed connector configuration name to "demo"

Mantis #0790320
    - Validated, Alert > inside 'Simulate Scenario' popup, the playbook execution is renamed 'Start Simulation'
    - Validated, Scenario > 'Run Scenario' renamed as simulate scenario
    - Validated execution of export scenario playbook
    - Validated execution of import scenario playbook

### CODE REVIEW

File: playbooks/SOC Simulator/Create Phishing Alert.json [hunk 1]
```diff
@@ -0,0 +1,218 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Create Phishing Alert",
```
Comment (cs-suyog-jinturkar) [inline playbooks/SOC Simulator/Create Phishing Alert.json]: Why we need this PB?

File: playbooks/SOC Simulator/Create Zero day Malicious File Indicator.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Create Zero day Malicious File Indicator",
+    "aliasName": null,
+    "tag": null,
```
Comment (cs-suyog-jinturkar) [inline playbooks/SOC Simulator/Create Zero day Malicious File Indicator.json]: Why we need this PB in SP? This should be part of some scenario SP, right?

Comment (cs-rohit-khune) [reply playbooks/SOC Simulator/Create Zero day Malicious File Indicator.json]: Hi @cs-suyog-jinturkar 

I have removed this playbook. Also made below changes

- Removed extra playbooks from the playbook collection
   > Create Alert
   > Create Phishing Alert
   > Create Zero day Malicious File Indicator
   > Get Malicious IP and URL
   > Replace Variables Values

- Updated Playbook collection name to "02 - Use Case - SOC Simulator"

- Updated info.json accordingly

File: playbooks/SOC Simulator/Create Phishing Alert.json (hunk 1 above)
Comment (cs-rohit-khune) [reply playbooks/SOC Simulator/Create Phishing Alert.json]: Hi @cs-suyog-jinturkar 

I have removed this playbooks also made below changes:

- Removed extra playbooks from the playbook collection
   > Create Alert
   > Create Phishing Alert
   > Create Zero day Malicious File Indicator
   > Get Malicious IP and URL
   > Replace Variables Values

- Updated Playbook collection name to "02 - Use Case - SOC Simulator"

- Updated info.json accordingly

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**